src/test/fuzz/util.h

```diff
@@ -577,15 +577,15 @@ class FuzzedSock : public Sock
 public:
     explicit FuzzedSock(FuzzedDataProvider& fuzzed_data_provider) : m_fuzzed_data_provider{fuzzed_data_provider}
     {
-          m_socket = fuzzed_data_provider.ConsumeIntegral<SOCKET>();
+          m_socket = fuzzed_data_provider.ConsumeIntegralInRange<SOCKET>(INVALID_SOCKET - 1, INVALID_SOCKET);
```

Comment:
Maybe this should receive similar treatment:

https://github.com/bitcoin/bitcoin/blob/03ecceedf6f15d2062e95b4533c5cea092c4c696/src/test/util/net.h#L78-L83

Reply:
Done!